source/ast/symbols/MemberSymbols.cpp

```diff
@@ -2196,7 +2196,6 @@ NetAliasSymbol& NetAliasSymbol::fromSyntax(const ASTContext& parentContext,
         if (!netType.isError()) {
             SmallVector<const IdentifierNameSyntax*> implicitNetNames;
             Expression::findPotentiallyImplicitNets(*expr, context, implicitNetNames);
-
```

Comment:
Random newline deletion seems unrelated to the rest of the diff

Reply:
fixed